### PR TITLE
Last second fixes

### PR DESCRIPTION
**Pull request by sylvinite**
This PR adds the following features:

- JSON is generated in runfolder when doing a normal analysis
- JSON now contains more QC data
- E-mailing from the nodes _might_ work now
- Novel alleles and Novel+Thresholds failed are now classed as Unknown
- Genologics config is now read from the microSALT config 

**How to test**:
On hasta:

0. If you have any files named *genologics* in your home directory. Temporarily rename them.
1. Pull branch into /home/proj/stage/bin/git/microSALT
2. us
3. source activate S_microSALT
4. export MICROSALT_CONFIG=/home/proj/dropbox/microSALT.json
4.1 doublecheck that the paths in /home/proj/dropbox/microSALT.json works for you
5. pip install .
6. microSALT analyse sample ACC5710A5 --email (email)
7. microSALT analyse project ACC5710 --email (email)

**Expected outcome**:

For steps 6 and 7 the following is expected to happen:
- An e-mail will be sent with a typing and qc report upon slurm completion. It is ok if it doesn't. If it doesnt, generate the e-mail with the `microSALT utils report` command.
- Sample A5, report sequence type as 'Unknown' in the typing report
- In the results folder for sample/project, there should be a json file. The contents of at least the QC report should be within it.

In other words, screenshots of the e-mailed typing report and the contents of the json file should be sufficient.

**Review:**
- [x] reviewed by self @sylvinite 
- [x] tests executed by @talnor 
- [x] deployed approved @ingkebil 

This is minor **version bump** because the code can't alter the results in a mysterious way; is backwards compatible, and will either outright crash or not generate the requested change; if it is incorrect.

### CODE REVIEW

**Comment by talnor** [comment]
Outcome of test:

- Email sent for both runs with typing and QC report. 
- Sequence type 'unkown' for both runs
- json files created in both project folders

The json files and typing reports:

project_microSALT_verification
sample_microSALT_verification


Screen Shot 2019-06-18 at 12 29 14 PM
Screen Shot 2019-06-18 at 12 44 28 PM

**Comment by ingkebil** [comment]
Merge ahead!
